src/Symfony/Component/VarDumper/Caster/DateCaster.php

```diff
@@ -79,4 +79,27 @@ public static function castTimeZone(\DateTimeZone $timeZone, array $a, Stub $stu
 
         return $filter & Caster::EXCLUDE_VERBOSE ? $z : $z + $a;
     }
+
+    /**
+     * Formats seconds.
+     *
+     * <code>
```

Comment:
no need for this code sample for a private method, tests are here to show examples